### ISSUE TITLE
ACC-269.1 (Level AA) _ Modify Modules _ _ Group skipping is not provided 

### ISSUE THREAD

**Issue by shubha1090** [kyma-project/busola#3153]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	High
Affected UI	-
Affected Check Point:	ACC-269.1
Short text:	ACC-269.1 (Level AA) _ Modify Modules _ _ Group skipping is not provided 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. Now Select ‘Modify Modules’ button which opens ‘Modules’ page.
•	Navigate to the side navigation menu button and press ‘F6’ key to skip group of complex Ui elements and observe the focus.

3. Issue Type: Manual
Observed Behavior:
The application does not support bypassing of larger elements and also there is no possibility to skip a complex UI element/group using keyboard throughout the application. 
Screenshot:
 

Expected Behavior:
The application should provide skip option for large and complex UI elements. There should be a mechanism available to bypass blocks of content that are repeated on multiple Web pages. A user must be able to locate the focus of a page as quickly as possible without, for example, having to traverse identical content after changing screens.
Note: 
This issue exists across the application and thus fix should be provided for all the occurrences.
________________________________________________________________________
Kindly refer to attached error and issue processing document for more details.
Regards,
Shubha

**Comment by shubha1090** [comment]
[ACC-269.1 (Level AA) _ Modify Modules _ _ Group skipping is not provided.docx](https://github.com/user-attachments/files/16660209/ACC-269.1.Level.AA._.Modify.Modules._._.Group.skipping.is.not.provided.docx)

[Issue processing-09.2021 (002).docx](https://github.com/user-attachments/files/16660210/Issue.processing-09.2021.002.docx)
